WordPress/Classes/ViewRelated/NUX/LoginSelfHostedViewController.swift

```diff
@@ -316,12 +316,17 @@ class LoginSelfHostedViewController: LoginViewController, NUXKeyboardResponder {
         }
     }
 
-
     @IBAction func handleForgotPasswordButtonTapped(_ sender: UIButton) {
         WordPressAuthenticator.openForgotPasswordURL(loginFields)
         WordPressAuthenticator.post(event: .loginForgotPasswordClicked)
     }
 
+    // MARK: - Dynamic type
+
+    override func didChangePreferredContentSize() {
+        super.didChangePreferredContentSize()
+        siteAddressLabel?.font = WPStyleGuide.fontForTextStyle(.body)
```

Comment:
Just out of curiosity:
Any idea why `.adjustsFontForContentSizeCategory = true` is not working and this is necessary?
Since it's a normal label with a preferred font, looks like it should 😮

Reply:
Nice catch, @etoledom !. I pushed another commit that sets `.adjusFontForContentSize` and removes explicit tracking of trait collections from that particular VC.